pkg/controller/controller_binding.go

```diff
@@ -434,7 +467,7 @@ func (c *controller) reconcileServiceInstanceCredential(binding *v1alpha1.Servic
 	// that's actually a soft delete-- i.e. we have some finalization to do.
 	if finalizers := sets.NewString(binding.Finalizers...); finalizers.Has(v1alpha1.FinalizerServiceCatalog) {
 		glog.V(4).Infof("Finalizing ServiceInstanceCredential %v/%v", binding.Namespace, binding.Name)
-		err = c.ejectServiceInstanceCredential(binding)
+		err := c.ejectServiceInstanceCredential(binding)
 		if err != nil {
 			s := fmt.Sprintf("Error deleting secret: %s", err)
 			glog.Warning(s)
```

Comment:
Prior to sending the first Unbind request to the Broker for a binding in orphan mitigation, we need to set the `OperationStartTime` field. It is not being set now because the `CurrentOperation` is not empty for a binding undergoing orphan mitigation.